### PR TITLE
Fix the HelloWorld example in the readme

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,10 @@ Instructions about how to build and install the Solidity compiler can be found i
 A "Hello World" program in Solidity is of even less use than in other languages, but still:
 
 ```
+pragma solidity ^0.4.16;
```
Comment (leonardoalt) [inline README.md:32]: Why 16?

Comment (axic) [reply README.md:32]: That's when `pure` was introduced.

**Review by leonardoalt — APPROVED** [review]

**Comment by chriseth** [comment]
This looks rather redundant now...

**Comment by axic** [comment]
That's what hello worlds are, pretty useless things 😉

**Pull request merged**